todo_app/data/session_items.py

```diff
@@ -54,6 +54,27 @@ def add_item(title):
     return item
 
 
+def delete_item(id):
+    """
+    Deletes the saved item with the specified ID.
+
+    Args:
+        id: The ID of the item.
+
+    Returns:
+        item: The saved item, or None if no items match the specified ID.
+    """
+    existing_items = get_items()
+    item = next((item for item in existing_items if item['id'] == int(id)), None)
+    print(item)
+    if (item != None):
+        print('remove')
```

Comment:
These look like debugging statements, not worth keeping around longer term - we should clear these out